frontend/src/actionCreators/parcelsActionCreator.spec.tsx

```diff
@@ -37,8 +33,15 @@ describe("fetchParcels action creator", () => {
     });
   });
 
-  it("Request successful, dispatches `success` with correct response", () => {
-    const params: IParcelListParams = { neLat: 1, neLong: 2, swLat: 3, swLong: 4 };
+  it('Request successful, dispatches `success` with correct response', () => {
+    const params: IParcelListParams = {
+      neLat: 1,
+      neLong: 2,
+      swLat: 3,
+      swLong: 4,
+      agencyId: null,
+      propertyClassificationId: null,
```

Comment:
To review: I had to add to new properties that were added after this test was written